src/common-principles.md

```diff
@@ -504,18 +504,19 @@ TSV files MUST be in UTF-8 encoding.
 
 Example:
 
-```Text
-onset   duration    response_time   trial_type        trial_extra
-200     20.0        15.8            word              中国人
-240     5.0         17.34e-1        visual            n/a
+```tsv {linenums="1"}
+onset	duration	response_time	trial_type	trial_extra
+200	20.0	15.8	word	中国人
+240	5.0	17.34e-1	visual	n/a
 ```
 
 !!! warning "Attention"
 
     The TSV examples in this document (like the one above this note) are occasionally
-    formatted using space characters instead of tabs to improve human readability.
-    Directly copying and then pasting these examples from the specification
-    for use in new BIDS datasets can lead to errors and is discouraged.
+    formatted with the addition of the row indexes
+    as first column.
+    Those indexes are presented for visual reference and
+    are not part of the tabular data file's content.
```

Comment:
I guess this admonition didn't get updated (like the `Text` block), since copying and pasting was made to work.

Reply:
I'll have a look.

Reply:
Okay, I reintroduced the advice of not copying and pasting, let me know if that addresses the issue (https://github.com/bids-standard/bids-specification/pull/2096/commits/e278984347a89d3c3bd13f9c2dd1fa047d1545ac)